guides/source/ruby_on_rails_guides_guidelines.md

```diff
@@ -106,9 +106,7 @@ The guides and the API should be coherent and consistent where appropriate. In p
 HTML Guides
 -----------
 
-Before generating the guides, make sure that you have the latest version of
-Bundler installed on your system. As of this writing, you must install Bundler
-1.3.5 or later on your device.
+Before generating the guides, make sure that you have the latest version of Bundler installed on your system. As of this writing, you must install Bundler 1.16.1 or later on your device.
```

Comment:
Might be somewhere you can link to for the version # of bundler too.

Reply:
Yeah, either rubygems or the releases section on github are the two ideas I have https://github.com/bundler/bundler/releases

Reply:
I added the link to rubygems. I like releases page as well. It was a tough choice. :)
I went with rubygems, as that's the version you're actually going to be installing with rubygem's `gem` CLI.